src/components/icons/index.ts

```diff
@@ -0,0 +1,7 @@
+export { DotsMenuIcon } from './DotsMenuIcon';
```

Comment:
I would take any feedback on whether this is a good idea to manage our icons as a "Library"